packages/components/src/styles/container/container.stories.ts

```diff
@@ -52,178 +49,112 @@ export const Default = {
-          colors: ['tranparent', 'tranparent', 'tranparent', 'transparent', 'rgb(var(--sd-color-primary, 0 53 142))']
-        }
-      },
-      args
-    })}`;
-  }
+  name: 'Variants',
+  render: () =>
+    html`<div class="grid grid-cols-2 gap-4">
+      <div class="sd-container default">
+        <div class="slot slot--border slot--text h-12">Default slot</div>
+      </div>
+
+      <div class="sd-container sd-container--variant-primary-100">
+        <div class="slot slot--border slot--text h-12">Default slot</div>
+      </div>
+
+      <div class="sd-container sd-container--variant-primary">
+        <div class="slot slot--border slot--text h-12">Default slot</div>
+      </div>
+
+      <div class="sd-container sd-container--variant-border-neutral-400">
+        <div class="slot slot--border slot--text h-12">Default slot</div>
+      </div>
+
+      <div class="sd-container sd-container--variant-white">
+        <div class="slot slot--border slot--text h-12">Default slot</div>
+      </div>
+    </div>`
 };
 
 /**
- * Use the `--padding-sm` class to adapt the container to smaller component widths.
+ * Use the `padding-sm` class to adapt the container to smaller component widths.
  */
 
 export const Padding = {
-  parameters: {
-    controls: {
-      exclude: ['sd-container--padding-...']
-    }
-  },
-  render: (args: any) => {
-    return html` ${generateTemplate({
-      axis: {
-        y: [
-          {
-            type: 'attribute',
-            name: 'sd-container--padding',
-            values: ['default', 'sd-container--padding-sm']
-          }
-        ]
-      },
-      args
-    })}`;
-  }
+  render: () =>
+    html`<div class="sd-container sd-container--padding-sm">
+      <div class="slot slot--border slot--text h-12">Default slot</div>
+    </div>`
 };
 
 /**
  * You can override the default padding values by setting your desired values as a style.
  */
 
 export const CustomPadding = {
-  parameters: {
-    controls: {
-      exclude: ['sd-container--padding-...']
-    }
-  },
-
-  render: (args: any) => {
-    return generateTemplate({
-      options: { templateContent: '<div class="%CLASSES%" style="padding:4rem;">%SLOT%</div>' },
-      args
-    });
-  }
+  render: () =>
+    html`<div class="sd-container" style="padding:4rem;">
+      <div class="slot slot--border slot--text h-12">Default slot</div>
+    </div>`
 };
 
 /**
- * You can add a triangle indentation to the container using the `--triangle-` class appended with one of the following positions 'top', 'right', 'bottom', 'left' (e.g. `sd-container--triangle-top`).
+ * You can add a triangle indentation to the container using the `triangle-class` appended with one of the following positions 'top', 'right', 'bottom', 'left' (e.g. sd-container--triangle-top).
+ *
+ * - A triangle can be shown to draw attention.
+ * - Triangle position can be `top`, `bottom`, `left` or `right`.
+ * - Default background option is white and can be overridden if desired.
```

Comment:
For the preview of Triangle position I think we should use the border neutral 400. What do you think @coraliefeil ?